lib/get.js

```diff
@@ -33,7 +33,11 @@ module.exports = opts => {
 				return decode(data.Item);
 			}
 			metrics.count(`${table}_status_4xx`);
-			throw new Error('URL not found');
+			return {
+				fromURL,
```

Comment:
is this some kind of ecma6 magic I didn't know about?  why not `fromUrl: fromUrl`?